src/test/java/org/folio/rest/impl/CirculationLogsImplTest.java

```diff
@@ -0,0 +1,84 @@
+package org.folio.rest.impl;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import io.restassured.http.Headers;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(OrderAnnotation.class)
+public class CirculationLogsImplTest extends TestBase {
+  private final Logger logger = LoggerFactory.getLogger(CirculationLogsImplTest.class);
+  private final String CIRC_LOGS_ENDPOINT = "/audit-data/circulation/logs";
+  private Headers headers = new Headers(tenant, perms, ctype);
+
+  @Test
+  @Order(1)
+  void initializeDatabaseAndLoadSampleData() {
```

Comment:
Looks like this method should be moved to @BeforeAll/@BeforeEach-method of TestBase/TestSuite. From my perspective this is mostly initialization step. + In the current implementation I can't move other tests one by one.